lib/spack/spack/build_systems/python.py

```diff
@@ -216,7 +216,7 @@ def view_file_conflicts(self, view, merge_map):
 
         return conflicts
 
-    def add_files_to_view(self, view, merge_map):
+    def add_files_to_view(self, view, merge_map, **kwargs):
```

Comment:
Noo, only do this if you're forwarding kwargs! You probably want to add `, fast=None` and then if you want to warn on the deprecated API then do so if `fast is not None`.

Reply:
I think "mysteriously eat any extra arguments" (which you rightly railed against just last week) is a much worse choice than "explicit keyword that's not used".